src/storage/store/node_column.cpp

```diff
@@ -15,103 +19,169 @@ using namespace kuzu::transaction;
 namespace kuzu {
 namespace storage {
 
-void FixedSizedNodeColumnFunc::readValuesFromPage(uint8_t* frame, PageElementCursor& pageCursor,
-    ValueVector* resultVector, uint32_t posInVector, uint32_t numValuesToRead) {
-    auto numBytesPerValue = resultVector->getNumBytesPerValue();
-    memcpy(resultVector->getData() + posInVector * numBytesPerValue,
-        frame + pageCursor.elemPosInPage * numBytesPerValue, numValuesToRead * numBytesPerValue);
-}
+struct FixedSizedNodeColumnFunc {
+    static void readValuesFromPage(uint8_t* frame, PageElementCursor& pageCursor,
```

Comment:
Can we get rid of `FixedSizedNodeColumnFunc `? Logically, it should be same as `ReadCompressedValuesFromPage` under the UNCOMPRESSED case.

Reply:
Yes. Not for InternalIDs yet, but it should be possible to get that working eventually.